File: src/main/java/de/tectoast/emolga/utils/draft/DraftPokemon.java
package de.tectoast.emolga.utils.draft;

public class DraftPokemon {

    public String name;
    public String tier;

    public DraftPokemon(String name, String tier) {
        this.name = name;
        this.tier = tier;
    }

    public String getName() {
        return name;
    }

    public String getTier() {
        return tier;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setTier(String tier) {
        this.tier = tier;
    }

    public DraftPokemon copy() {
        return new DraftPokemon(name, tier);
    }

    @Override
    public String toString() {
        return "DraftPokemon{" +
                "name='" + name + '\'' +
                ", tier='" + tier + '\'' +
                '}';
    }
}

File: src/main/kotlin/de/tectoast/emolga/utils/draft/DraftPokemon.kt
package de.tectoast.emolga.utils.draft

import de.tectoast.emolga.utils.json.Config
import kotlinx.serialization.Serializable

@Serializable
@Config("DraftPokemon", "Ein gepicktes Pokemon")
data class DraftPokemon(
    @Config("Name", "Der Name des Pokemons")
    var name: String = "NAME",
    @Config("Tier", "Das Tier des Pokemons")
    var tier: String = "TIER",
    @Config("Free", "Ob das Pokemon ein Freepick ist")
    var free: Boolean = false,
    var quit: Boolean = false,
    var noCost: Boolean = false
)
